src/com/facebook/buck/android/dalvik/DalvikStatsCache.java

```diff
@@ -33,8 +33,9 @@
 
   DalvikStatsTool.Stats getStats(FileLike entry) {
     String name = entry.getRelativePath();
-    if (!name.endsWith(".class")) {
+    if (!name.endsWith(".class") || name.endsWith("module-info.class")) {
       // Probably something like a pom.properties file in a JAR: this does not contribute
+      //skipping special class files like module descriptor - here no classes will be declared and class visitor thows error
```

Comment:
Could you move your comment 1 line down and add 1 space after "//"? Thanks @surapuramakhil

Reply:
okay :)

Reply:
The problem is that you inserted your comment in the middle of another comment and a wrong spelling word "throws". The comment should look like:
```
      // Probably something like a pom.properties file in a JAR: this does not contribute
      // to the linear alloc size, so return zero.
      // skipping special class files like module descriptor - here no classes will be
      // declared and class visitor throws error
```

Reply:
Also, should we use name.equals() instead of name.endsWith()?

Reply:
No @v-jizhang  , name is taken as relative path.

`String name = entry.getRelativePath();`

Reply:
What if there is such a class FooBar-module-info.class we don't want to exclude?

Reply:
Done